prebid-mobile/modules/rendering/android-sdk-integration-pb.md

```diff
@@ -212,9 +224,9 @@ Pay attention that the `loadAd()` should be called on the main thread.
 #### Step 1: Create a Rewarded Ad Unit
 {:.no_toc}
 
-Create the `RewardedAdUnit` object with parameters:
+Create the `RewardedAdUnit` object with the parameters:
 
-* `adUnitId` - an ID of Stored Impression on the Prebid server.
+* `adUnitId` - this is an arbitrary descriptor for the ad. You might use it as a descriptive name in targeting or for reporting.  For example you might name it "account details screen ad" or "home page ad name."
```

Comment:
suggest replacing this with "this is the value of the adunit name in the ad server. e.g. for GAM, it's the adunit name. Other ad servers may call this something different, but the idea is that the buyers want to know which slot in the app is up for auction for targeting and reporting. An example would be "/home/upper-mobile-banner".